docker/Dockerfile

```diff
@@ -1,20 +1,19 @@
 # This dockerfile is based on Bitrise but with a lot of extra cruft we don't need removed
 FROM ubuntu:focal
-MAINTAINER Parity Technologies <admin@parity.io>
+LABEL maintainer="Parity Technologies <admin@parity.io>"
 
 ENV ANDROID_SDK_ROOT /opt/android-sdk-linux
 ENV ANDROID_HOME /opt/android-sdk-linux
 ENV NDK_HOME /opt/android-ndk
-ENV ANDROID_NDK_VERSION r23b
 ENV JAVA_HOME /usr/lib/jvm/java-11-openjdk-amd64/
 
 RUN apt-get -y update && apt-get -y install default-jdk wget unzip curl clang
+RUN apt-get -y install python git
```

Comment:
Combine it into the one layer, e.g. `RUN apt-get -y update && apt-get -y install default-jdk wget unzip curl clang python git`

Reply:
Here, when I do so, I get a strange behavior while installing jdk. It prompts to chose a region and while building an image stuck at this point. When it separate, it goes without asking for it. Don't know exactly why it behaves like this.

Reply:
` ENV DEBIAN_FRONTEND=noninteractive` solves this